Improve python software installations (#265)
greenbone/docs: src/22.4/source-build/prerequisites.rst

  echo "8AE4BE429B60A59B311C2E739823FAA60ED1E580:6:" > /tmp/ownertrust.txt
  gpg --import-ownertrust < /tmp/ownertrust.txt

Setup Python
------------

> To allow loading the to be installed Python packages the installation path must

Reviewer: ```suggestion
To allow the downloading of the Python packages, the installation path must
```
